http/vertx-web-client/src/test/java/io/quarkus/ts/http/vertx/webclient/PureVertxHttpClientTest.java

```diff
@@ -25,11 +26,13 @@ public class PureVertxHttpClientTest {
     URL url;
 
     WebClient httpClient;
+    Vertx vertx;
     static final int EXPECTED_EVENTS = 25000;
     private static final int TIMEOUT_SEC = 10;
 
     @BeforeEach
     public void setup() {
+        vertx = Vertx.vertx();
         httpClient = WebClient.create(Vertx.vertx(), new WebClientOptions());
```

Comment:
on the line above, you assing `vertx = Vertx.vertx();` and here you again create brand new Vert.x instance that won't be closed. please use here vertx var

Reply:
`WebClient.create(Vertx.vertx())`